src/index.html

```diff
@@ -8,9 +8,42 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
+    <link rel="icon" href="images/Search.svg">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      data-qa="big"
+      class="form"
+      action="#"
+      method="get"
+    >
+      <img class="dandruff" src="images/Search.svg" alt="search">
+      <label for="search"></label>
+      <input
+        data-qa="keypress"
+        name="search"
+        id="search"
+        type="text"
+        placeholder="Try &quot;Los Angeles&quot;"
+        class="search"
+      >
+    </form>
+    <form
+      data-qa="small"
+      class="form--small"
+      action="#"
+      method="get"
+    >
+      <img class="dandruff--small" src="images/Search.svg" alt="search--small">
+      <label for="search--small"></label>
+      <input
+        name="search--small"
+        id="search--small"
+        type="text"
+        placeholder="Try &quot;Los Angeles&quot;"
+        class="search--small"
```

Comment:
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/99403198/abfdb1bd-784e-4a64-994c-1f8c0f213096)
```suggestion
        class="search search--small"
```